requirements.txt

```diff
@@ -1 +1 @@
-typeguard
+typeguard==2.7.1
```

Comment:
Seeing as this requirements file gets added to the pip dependencies, should we relax this to be `typeguard>=`? I worry that this will create conflicts with other packages that utilize typeguard.

Reply:
I totally agree. Can I do that in another PR? I'm going to relax the requirements on the minor version for each dependency too. Ex pip install tensorflow~=2.1